client/signup/main.jsx

```diff
@@ -779,6 +782,21 @@ class Signup extends Component {
 		}
 	}
 
+	shouldShowZendeskPresalesChat() {
+		const isEnglishLocale = config( 'english_locales' ).includes( getLocaleSlug() ?? '' );
+		const currentTime = new Date();
+
+		return (
+			! this.props.isLoggedIn &&
+			( isCurrentPlanFlow( this.props.flowName ) || this.props.flowName === 'onboarding' ) &&
+			isEnglishLocale &&
+			currentTime.getUTCHours() >= 15 &&
```

Comment:
I think we can also check for the presence of `recognized_logins` cookie and hide the widget if it exists, so that we're not getting chat requests from existing logged-out users.

Reply:
> Isn't this date/time check already done as part of the happychat.user.availability?.presale_zendesk

Yes, but the flag isn't actually used here (because it's not available in signup)